apps/api_web/lib/api_web/legacy_stops.ex

```diff
@@ -0,0 +1,263 @@
+      "Malden Center" => ["WR-0045-S"],
+      "Manchester" => ["GB-0254", "GB-0254-01", "GB-0254-02"],
+      "Mansfield" => ["NEC-2040", "NEC-2040-01", "NEC-2040-02"],
+      "Melrose Cedar Park" => ["WR-0067", "WR-0067-01", "WR-0067-02"],
+      "Melrose Highlands" => ["WR-0075", "WR-0075-01", "WR-0075-02"],
+      "Middleborough/ Lakeville" => ["MM-0356-S"],
+      "Mishawum" => ["NHRML-0116", "NHRML-0116-01", "NHRML-0116-02"],
+      "Montello" => ["MM-0186", "MM-0186-CS", "MM-0186-S"],
+      "Montserrat" => ["GB-0198", "GB-0198-01", "GB-0198-02"],
+      "Morton Street" => ["DB-2230", "DB-2230-01", "DB-2230-02"],
+      "Nantasket Junction" => ["GRB-0183-S"],
+      "Natick Center" => ["WML-0177", "WML-0177-01", "WML-0177-02"],
+      "Needham Center" => ["NB-0127-S"],
+      "Needham Heights" => ["NB-0137-S"],
+      "Needham Junction" => ["NB-0120-S"],
+      "Newburyport" => ["ER-0362", "ER-0362-01", "ER-0362-02"],
+      "Newmarket" => ["DB-2265", "DB-2265-01", "DB-2265-02"],
+      "Newtonville" => ["WML-0081-02"],
+      "Norfolk" => ["FB-0230-S"],
+      "North Beverly" => ["ER-0208", "ER-0208-01", "ER-0208-02"],
+      "North Billerica" => ["NHRML-0218", "NHRML-0218-01", "NHRML-0218-02"],
+      "North Leominster" => ["FR-0451", "FR-0451-01", "FR-0451-02"],
+      "North Scituate" => ["GRB-0233-S"],
+      "North Station" => ["BNT-0000"],
+      "North Station-01" => ["BNT-0000-01"],
+      "North Station-02" => ["BNT-0000-02"],
+      "North Station-03" => ["BNT-0000-03"],
+      "North Station-04" => ["BNT-0000-04"],
+      "North Station-05" => ["BNT-0000-05"],
+      "North Station-06" => ["BNT-0000-06"],
+      "North Station-07" => ["BNT-0000-07"],
+      "North Station-08" => ["BNT-0000-08"],
+      "North Station-09" => ["BNT-0000-09"],
+      "North Station-10" => ["BNT-0000-10"],
+      "North Wilmington" => ["WR-0163-S"],
+      "Norwood Central" => ["FB-0148", "FB-0148-01", "FB-0148-02"],
+      "Norwood Depot" => ["FB-0143", "FB-0143-01", "FB-0143-02"],
+      "place-dudly" => ["place-nubn"],
+      "Plimptonville" => ["FB-0177-S"],
+      "Plymouth" => ["PB-0356-S"],
+      "Porter Square" => ["FR-0034", "FR-0034-01", "FR-0034-02"],
+      "Prides Crossing" => ["GB-0222", "GB-0222-01", "GB-0222-02"],
+      "Providence" => ["NEC-1851", "NEC-1851-01", "NEC-1851-02", "NEC-1851-03", "NEC-1851-05"],
+      "Quincy Center" => ["MM-0079-S"],
+      "Reading" => ["WR-0120-S"],
+      "Readville" => ["DB-0095", "FB-0095-04", "FB-0095-05", "NEC-2192-02", "NEC-2192-03"],
+      "River Works / GE Employees Only" => ["ER-0099", "ER-0099-01", "ER-0099-02"],
+      "Rockport" => ["GB-0353-S"],
+      "Roslindale Village" => ["NB-0064-S"],
+      "Route 128" => ["NEC-2173", "NEC-2173-01", "NEC-2173-02"],
+      "Rowley" => ["ER-0312-S"],
+      "Ruggles" => ["NEC-2265"],
+      "Ruggles-01" => ["NEC-2265-01"],
+      "Ruggles-02" => ["NEC-2265-02"],
+      "Ruggles-03" => ["NEC-2265-03"],
```

Comment:
I noticed that, while our previous back-compat code expanded `Back Bay`, `North Station`, and `South Station` to their corresponding platformed stops like `Back Bay-01`, we never did the same for `Ruggles`. Did these platformed stops always exist, unlike the others?